qemu/tests/cfg/seabios.cfg

```diff
@@ -8,19 +8,54 @@
     boot_menu = on
     enable_sga = yes
     image_verify_bootable = no
-    boot_menu_key = "f12"
-    Host_RHEL.m7:
-        boot_menu_key = "esc"
-    boot_menu_hint = "Press .*(F12|ESC) for boot menu"
-    # Specify the boot device name which you want to test here.
-    boot_device = "iPXE"
-    Host_RHEL.m6:
-        boot_device = "gPXE"
-    # SGA Bios info message, using sep as ";"
-    # Please update this message to suit for your own system.
-    restart_key = "ctrl-alt-delete"
-    Host_RHEL:
-        sgabios_info = "Google, Inc.\s"
-        sgabios_info += "Serial Graphics Adapter .*\s"
-        sgabios_info += "SGABIOS \$Id: sgabios.S .*\s"
-        sgabios_info += "\d \d\s"
+    variants:
+        - reboot_time_test:
+            no q35
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_format_stg = "qcow2"
+            drive_format_stg = virtio
+            image_size_stg = 4G
+            create_image_stg = yes
+            remove_image_stg = yes
+            bootindex_stg = 0
+            bootindex_image1 = 1
+            variants:
+                - time0:
+                    boot_reboot-timeout = 0
+                - time1:
+                    boot_reboot-timeout = 1000
+                - time2:
+                    boot_reboot-timeout = 10000
+                - time3:
+                    boot_reboot-timeout = 30000
+                - time4:
+                    boot_reboot-timeout = 60000
+                - time5:
+                    boot_reboot-timeout = 65535
+                - time6:
+                    boot_reboot-timeout = 65536
+                - time7:
+                    boot_reboot-timeout = -1
+        - sga_restart:
+            boot_menu_key = "f12"
+          # SGA Bios info message, using sep as ";"
+          # Please update this message to suit for your own system.
+            restart_key = "ctrl-alt-delete"
+            Host_RHEL:
+                sgabios_info = "Google, Inc.\s"
+                sgabios_info += "Serial Graphics Adapter .*\s"
+                sgabios_info += "SGABIOS \$Id: sgabios.S .*\s"
+                sgabios_info += "\d \d\s"
+            Host_RHEL.m7:
+                boot_menu_key = "esc"
+            boot_menu_hint = "Press .*(F12|ESC) for boot menu"
+           # Specify the boot device name which you want to test here.
+            boot_device = "iPXE"
+            Host_RHEL.m6:
+                boot_device = "gPXE"
+          # SGA Bios info message, using sep as ";"
+          # Please update this message to suit for your own system.
+            restart_key = "ctrl-alt-delete"
```

Comment:
L57-L59 is same as L42-L44. Please recheck it.

Reply:
removed L57-L59